### DISCUSSION TITLE
vscode extension error

### DISCUSSION THREAD

**Discussion by mousezheng** [community/community#19740] [Copilot Conversations]
In vscode `Failed to save 'XXX.go': The content of the file is newer. Please compare your version with the file contents or overwrite the content of the file with your changes.`

I guess the extension（`GitHub Copilot v1.31.6194`） may coincide with keyboard input when writing files

**Comment by huabin** [comment]
I think it's a conflict in the version of the file `xxx.go`.

**Reply by mousezheng** [reply]
That's right, but it's not a solution

**Reply by huabin** [reply]
Here are a few ways to resolve this conflict

1. Resolve this conflict with the command
2. Resolve via VSCode. Open the conflict file, if I remember correctly, there is a conflict flag in the top right corner, click it, and select the correct version.

Hope this can help you.

**Comment by abdulwahidkahar** [comment]
> Failed to save 'XXX.go': The content of the file is newer. Please compare your version with the file contents or overwrite the content of the file with your changes

VSCode will never refresh the file if you have changes in that file that are not saved to disk. However, if the file is open and does not have changes, it will replace with the changes on disk, that is true.

There is currently no way to disable this behaviour.
